packages/wallet/src/services/util/persistentTrackerSubjects.ts

```diff
@@ -28,6 +29,11 @@ export class PersistentCollectionTrackerSubject<T> extends TrackerSubject<T[]> {
         })
       )
     );
+    this.store = store;
+  }
+  next(value: T[]): void {
+    this.store && this.store.setAll(value);
```

Comment:
If we're doing this in .next(), maybe we can:
- remove `tap(store.setAll.bind(store))` in the constructor
- get rid of [unspendableUtxoSubject$](https://github.com/input-output-hk/cardano-js-sdk/pull/228/files#diff-b2b947493b30368269c8ae9c56fb7723a623e8c10f22c17a4d8da41bda21e33bR49) (use [NEVER](https://rxjs.dev/api/index/const/NEVER) instead) and in `setUnspendable` just call `.next` on persistent subject directly